docs/writing-stories/decorators.md

```diff
@@ -38,26 +38,34 @@ Some components require a “harness” to render in a useful way. For instance,
 
 ## “Context” for mocking
 
-Some libraries require components higher up in the component hierarchy to render correctly. For example, in Styled Components, a `ThemeProvider` is necessary if your components use themes. Add a single global decorator that adds this context to all stories in [`.storybook/preview.js`](../configure/overview.md#configure-story-rendering):
+Framework-specific libraries (e.g., React's Styled Components, Vue's Fortawesome) may require additional configuration to render correctly in Storybook.
```

Comment:
```suggestion
Framework-specific libraries (e.g., [Styled Components](https://styled-components.com/) for React, Fortawesome for Vue) may require additional configuration to render correctly in Storybook.
```

I couldn't find "Fortawesome for Vue". I found "Fontawesome for Vue", but it didn't seem to require a Provider, so I'm not sure of your intent here.

Reply:
Oh, wait. Is this it? https://vuecomponent.com/integrations/fontawesome.html#_3-or-add-to-library-to-use-across-the-whole-application